service/k8sclient/k8sclient_test.go

```diff
@@ -75,10 +79,25 @@ func TestK8sClient(t *testing.T) {
 		})
 
 		t.Run("All pods are ready", func(t *testing.T) {
+			var xtradbCluster pxc.PerconaXtraDBCluster
+			err := client.kubeCtl.Get(ctx, string(perconaXtraDBClusterKind), name, &xtradbCluster)
+			require.NoError(t, err)
+
+			expected_ready := xtradbCluster.Status.HAProxy.Ready
+			expected_total := xtradbCluster.Status.HAProxy.Size
+			expected_ready += xtradbCluster.Status.PMM.Ready
+			expected_total += xtradbCluster.Status.PMM.Size
+			expected_ready += xtradbCluster.Status.ProxySQL.Ready
+			expected_total += xtradbCluster.Status.ProxySQL.Size
+			expected_ready += xtradbCluster.Status.PXC.Ready
+			expected_total += xtradbCluster.Status.PXC.Size
+
 			cluster, err := getXtraDBCluster(ctx, client, name)
 			require.NoError(t, err)
-			assert.Equal(t, cluster.DetailedState.CountAllPods(), cluster.DetailedState.CountReadyPods())
+			assert.Equal(t, expected_ready, cluster.DetailedState.CountReadyPods())
```

Comment:
I expected to have just an integer like `2` here and usage `Equal` instead of `Greater`

Reply:
I had it like this initially, 2 for xtradb, 9 for mongo. Didn't work for mongo in CI, worked in local test. Let me try it again, maybe I just missed something.

Reply:
@BupycHuk It seems to be ok now.